adal/proguard-rules.pro

```diff
@@ -1,11 +1,8 @@
-# To enable ProGuard in your project, edit project.properties
-# to define the proguard.config property as described in that file.
-#
 # Add project specific ProGuard rules here.
```

Comment:
I know what proguards does, It's fine and nice to use it, most apps turn on proguard for release build. But we don't have any rules in the proguard file.